src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/document/DocumentManagementService.java

```diff
@@ -0,0 +1,142 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.services.document;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.core.io.Resource;
+import org.springframework.hateoas.hal.HalLinkDiscoverer;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.bulkscanprocessor.exceptions.UnableToUploadDocumentException;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.document.output.FileUploadResponse;
+import uk.gov.hmcts.reform.bulkscanprocessor.services.document.output.Pdf;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.StreamSupport.stream;
+import static org.springframework.http.MediaType.MULTIPART_FORM_DATA_VALUE;
+
+@Service
+public class DocumentManagementService {
+
+    private static final String MULTIPART_FORM_PARAM = "files";
+    private static final String CONTENT_TYPE = "Content-Type";
+    private static final String CLASSIFICATION = "classification";
+    private static final String DOCUMENTS_PATH = "/documents";
+    private static final String RESTRICTED = "RESTRICTED";
```

Comment:
Both `CLASSIFICATION` and `RESTRICTED` are only used once, and their names don't clarify what those constants are for.

I think it would be much clearer to just inline them (use strings where the constant is used)